packages/design-picker/src/components/design-picker-category-filter/style.scss

```diff
@@ -5,6 +5,41 @@
 $design-picker-category-filter-text-color: #3c434a;
 $design-picker-category-filter-text-color-active: var(--studio-gray-100);
 
+.design-picker {
+	.design-picker__filters {
+		display: flex;
+		justify-content: space-between;
+
+		.design-picker__category-filter {
+			width: 100%;
+		}
+
+		.design-picker__design-your-own-button {
+			flex-shrink: 0;
+			height: 32px;
+			line-height: 14px;
+			margin-top: 9px;
+			margin-left: 10px;
+		}
+
+		.design-picker__design-your-own-button-without-categories {
+			margin-left: auto;
+			margin-top: -60px;
+		}
+	}
+
+	&.design-picker__unified {
+		.responsive-toolbar-group__dropdown {
+			&.design-picker__category-filter {
+				.responsive-toolbar-group__grouped-list {
+					justify-content: flex-start;
+					width: 100%;
```

Comment:
Can we remove the styles below now?

https://github.com/Automattic/wp-calypso/blob/f99076e25b56a050f2826d8f0d8ee966dd8457b0/packages/design-picker/src/components/style.scss#L396-L398

Reply:
Sure, removed.